bot/exts/evergreen/latex.py

```diff
@@ -0,0 +1,75 @@
+import asyncio
+import re
+from concurrent.futures import ThreadPoolExecutor
+from io import BytesIO
+
+import discord
+import matplotlib.pyplot as plt
+from discord.ext import commands
+
+# configure fonts and colors for matplotlib
+plt.rcParams.update(
+    {
+        "font.size": 16,
+        "mathtext.fontset": "cm",  # Computer Modern font set
+        "mathtext.rm": "serif",
+        "figure.facecolor": "38383F",  # matches Discord's dark mode background color
```

Comment:
```suggestion
        "figure.facecolor": "36393F",  # matches Discord's dark mode background color
```

Nitpicking, but this is the exact colour code for dark mode, can be found in `discord.colour.Colour.dark_theme`